API/resources/resources.json

```diff
@@ -52,10 +57,14 @@
           "dst": "%{tizenrt}/build/configs/artik053/jerryscript/"
         },
         {
-          "condition": "'%{app}' == 'jerryscript'",
+          "condition": "'%{app}' == 'jerryscript' and '%{build-type}' == 'release'",
           "src": "%{config}/tizenrt-jerryscript.config",
           "dst": "%{tizenrt}/build/configs/artik053/jerryscript/defconfig"
-
+        },
+        {
+          "condition": "'%{app}' == 'jerryscript' and '%{build-type}' == 'debug'",
+          "src": "%{config}/tizenrt-jerryscript-debug.config",
+          "dst": "%{tizenrt}/build/configs/artik053/jerryscript/defconfig"
```

Comment:
Same as above.

Reply:
@rtakacs : Thanks, it was a good idea. I've removed the condition also (It was needless after replacing the application name and build-type).